### PR TITLE
Add net.rpcs3.RPCS3

### PR DESCRIPTION
**Pull request by Lctrs**

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build net.rpcs3.RPCS3

**Comment by Lctrs** [comment]
@hfiguiere Can you cancel the test build ? I forgot to add the flathub.json file.

**Comment by bilelmoussaoui** [comment]
> @hfiguiere Can you cancel the test build ? I forgot to add the flathub.json file.

Done

**Comment by Lctrs** [comment]
Thank you. I've included it now. It's only a `x86_64` build. You can rerun it.

**Comment by hfiguiere** [comment]
sadly I can't but @bilelmoussaoui did :-)

**Comment by hfiguiere** [comment]
@Lctrs I think you can by posting the command I did.

**Comment by Lctrs** [comment]
bot, build net.rpcs3.RPCS3

File: net.rpcs3.RPCS3.yaml [hunk 1]
```diff
@@ -0,0 +1,159 @@
+id: "net.rpcs3.RPCS3"
```
Comment (eszlari) [inline net.rpcs3.RPCS3.yaml:1]: Do you really want to keep all that quoting? Isn't YAML all about being less noisy?

Comment (Lctrs) [reply net.rpcs3.RPCS3.yaml:1]: I prefer to be explicit on the types of values. I've had enough bad experiences with this with YAML.

**Comment by Lctrs** [comment]
bot, build net.rpcs3.RPCS3

**Comment by Lctrs** [comment]
bot, build net.rpcs3.RPCS3

**Comment by nedrichards** [comment]
Is there an upstream request for this?

**Comment by Lctrs** [comment]
@LeandroStanger opened https://github.com/RPCS3/rpcs3/issues/5935 a year ago. Upstream said they're happy with AppImage, I don't think their position has evolved since.

**Comment by nedrichards** [comment]
bot, build net.rpcs3.RPCS3

**Pull request closed without merging**